pages/dkp/kaptain/1.2.0-1.0.0/version-policy/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: layout.pug
+navigationTitle: Version Support Policy
+title: Version Support Policy
+menuWeight: 5
+excerpt: Kaptain's supported version policy
+beta: false
+enterprise: false
+---
+
+D2iQ supports `N-2` of the latest `MAJOR.MINOR` release of Kaptain.
+With the GA release of Kaptain 1.0.0, support continues for Kaptain 1.0, 0.6, and 0.5, including all patch releases.
+Support for Kaptain 0.4.0 expires.
+
+Users should upgrade their Kaptain installations clusters with every new release to stay up to date with the latest features and bug fixes.
+
+## Supported Konvoy Versions
+
+Kaptain 1.0.0 runs on Konvoy 1.7.
+
+## Supported Kommander Versions
+
+Kaptain 1.0.0 is not yet supported on Kommander.
+
+Note that it is possible to use [Kommander's Dex instance](../custom-configuration/external-dex) for authenticating users in Kaptain.
```

Comment:
These two sentences seem to contradict each other.  Please clarify what we mean by 'not yet supported'?   Is that you cannot install Kaptain using Kommander?  Or something else?

Reply:
You cannot install Kaptain from Kommander but you can use Kommander's Dex instance for authentication.